Add timeout and retries settings to integratord configuration (#6442)
wazuh/wazuh-documentation: source/user-manual/manager/manual-integration.rst

The full configuration reference for the Integrator daemon can be found :ref:`here <reference_ossec_integration>`.

Optional settings
^^^^^^^^^^^^^^^^^

> The `Integrator` daemon uses the `optional settings` fields to configure the internal behavior of the integrations.

Reviewer: ```suggestion
The `Integrator` daemon uses the `optional settings` fields to tailor the integration behavior.
```

> The full configuration reference for the Integrator daemon can be found :ref:`here <reference_ossec_integration>`.

Reviewer: ```suggestion
You can find the full configuration reference for the Integrator daemon in the :ref:`Integration <reference_ossec_integration>` section of the reference guide.
```

   .. note::

>       You can set a JSON object with customization fields using the options tag. Visit the `Slack API reference <https://api.slack.com/reference/messaging/attachments#legacy_fields>`__ for information about available customization fields.

Reviewer: ```suggestion
      You can set a JSON object with customization fields using the ``options`` tag. Visit the `Slack API reference <https://api.slack.com/reference/messaging/attachments#legacy_fields>`__ for information about available customization fields.
```

   .. note::

>       You can set a JSON object with customization fields using the options tag. Visit the `PagerDuty API reference <https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTgx-send-an-alert-event>`__ for information about available customization fields.

Reviewer: ```suggestion
      You can set a JSON object with customization fields using the ``options`` tag. Visit the `PagerDuty API reference <https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTgx-send-an-alert-event>`__ for information about available customization fields.
```

   .. note::

>       You can set a JSON object with customization fields using the options tag. Visit the `Shuffle API reference <https://shuffler.io/docs/API>`__ for information about available customization fields.

Reviewer: ```suggestion
      You can set a JSON object with customization fields using the ``options`` tag. Visit the `Shuffle API reference <https://shuffler.io/docs/API>`__ for information about available customization fields.
```
